happy-travel/edo
# Feature/aa 1871 add child agencies payment settings

Issue: https://github.com/happy-travel/agent-app-project/issues/1871

Test cases:
1) Create new root agency. Check if system settings and payment settings setted well.
2) Invite child agency. Check if contract kind and settings inherited well.
3) Invite agent to child agency. Check if payment settings inherited well.
4) Check if child agency's and child agent's payment settings can be updated restricted by root agency's payment settings
5) Check if payment settings affect to third search step (restricts allowed payment methods)
6) Check if root agencies contract kind change affects to child agencies and his agent's payment settings
